### PR TITLE
Small PR updating some broken bits of the docs

### PR DESCRIPTION
**Pull request by ericdill**

### CODE REVIEW

File: sphinx/source/docs/user_guide/widgets.rst [hunk 1]
```diff
@@ -10,7 +10,13 @@ Bokeh Widgets
 
 Bokeh provides a number of widgets to create interactive data-backed dashboards.
 
-There are several examples available: `glyphs <https://github.com/bokeh/bokeh/blob/master/examples/glyphs/glyphs.py>`_, `widgets server <https://github.com/bokeh/bokeh/blob/master/examples/glyphs/widgets_server.py>`_,  `table server <https://github.com/bokeh/bokeh/blob/master/examples/glyphs/table_server.ipynb>`_, `app reveal <https://github.com/bokeh/bokeh/blob/master/examples/embed/app_reveal.py>`_ and `population <https://github.com/bokeh/bokeh/blob/master/examples/glyphs/population_server.py>`_. You can use these examples as a starting point for your projects.
+There are several examples available:
+`glyphs <https://github.com/bokeh/bokeh/blob/master/examples/glyphs/glyphs.py>`_,
+`widgets server <https://github.com/bokeh/bokeh/blob/master/examples/glyphs/widgets_server.py>`_,
+`table server <https://github.com/bokeh/bokeh/blob/master/examples/glyphs/table_server.ipynb>`_,
```
Comment (ericdill) [inline sphinx/source/docs/user_guide/widgets.rst]: This link is broken. I'm not exactly sure what examples/glyphs/table_server.ipynb is supposed to point to as it is no longer appears to be in the codebase. Perhaps it is supposed to be examples/glpyhs/data_tables.ipynb?

Comment (bryevdv) [inline sphinx/source/docs/user_guide/widgets.rst]: @ericdill that would be a fine link to include there.

**Comment by ericdill** [comment]
@bryevdv Updated that link and squashed the commits down to 1. Should be good to go

**Comment by bryevdv** [comment]
Hi @ericdill thanks I very much appreciate the PR, however I am in the middle of rewriting the entire UG from scratch, so if we merge this the changes will just be blown away when the other PR hits (and also probably have a merge conflict too). For those reasons, I am going to close this PR as "superseded". I would very much like feedback on the new UG which is intended to be complete for the 0.9 release. The PR is here #2207 and a test deploy of the WIP is here

http://bokeh.pydata.org/en/test/docs/user_guide.html

**Comment by ericdill** [comment]
Ah, sorry it took me a while to get back to this PR.  I'll go have a look at the rewrite

**Pull request closed without merging**